src/main/webapp/index.html

```diff
@@ -4,8 +4,27 @@
     <meta charset="UTF-8">
     <title>Student Recommendations</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/styles/fonts.css">
+    <link rel="stylesheet" href="/styles/common.css">
   </head>
   <body>
-    <h1>Welcome to Rania and Lorna's capstone project</h1>
+    <div class="header">
+      <h1>StudentRecs</h1>
+      <h3>a recommendation system for students, by students!</h3>
+
+      <nav>
+        <ul>
+          <li><a class="active" href="#recmap">Recommendation Map</a></li>
+          <li><a href="#user">User Directory</a></li>
+        </ul>
+      </nav>
+    </div>
+    <div class="content">
+      <h1>TODO: add content</h1>
+    </div>
+    <footer class="footer">
+      <h2>made by Rania Rusly and Lorna Armstrong</h2>
+      <h3>STEP intern 2020</h3>
```

Comment:
Umm.. these are not "headers", right? It's true that having `h2` or `h3` here gives us the desired font size and look. But if it's not actually a "heading", I will ask to use normal `div`/`p`  elements here and do the styling using CSS properties.